Src/AutoFixture.NUnit3.UnitTest/FrozenAttributeIntegrationTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture(Description="This test suite is not exhaustive")]
+    public class FrozenAttributeIntegrationTest
+    {
+        [Theory, AutoData]
+        public void FrozenWorksCorrectly(
+            [Frozen]int anyInteger, 
+            [Frozen]int anyOtherInteger,
+            [Frozen] DateTime anyDateTime,
+            [Frozen] DateTime anyOtherDateTime,
+            [Frozen] CultureInfo anyCultureInfo,
+            [Frozen] CultureInfo anyOtherCultureInfo)
+        {
+            Assert.That(anyInteger, Is.EqualTo(anyOtherInteger));
+            Assert.That(anyDateTime, Is.EqualTo(anyOtherDateTime));
+            Assert.That(anyCultureInfo, Is.EqualTo(anyOtherCultureInfo));
```

Comment:
Doesn't this assertion pass even if you remove the `[Frozen]` attributes from both the arguments in question?